src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,124 @@
+package wolox.training.models;
+
+
+import javax.persistence.Id;
+
+import javax.persistence.*;
+
+@Entity()
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    @Column(nullable = true)
+    private String genre;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column(nullable = false)
+    private String image;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String subtitle;
+
+    @Column(nullable = false)
+    private String publisher;
+
+    @Column(nullable = false)
+    private String year;
+
+    @Column(nullable = false)
+    private Integer pages;
+
+    @Column(nullable = false)
+    private String isbn;
```

Comment:
We can improve the validation of required fields by using the `@ NotNull` annotation. Thus, we do not need to reach the persistence layer to interrupt execution when the values are null.